.github/workflows/cla-assistant.yaml

```
@@ -0,0 +1,47 @@
+name: "CLA Assistant"
+on:
+  issue_comment:
+    types: [created]
+  pull_request_target:
+    types: [opened, closed, synchronize]
+# explicitly configure permissions, in case your GITHUB_TOKEN workflow permissions are set to read-only in repository settings
+permissions:
+  actions: read
+  contents: read # this can be 'read' if the signatures are in remote repository
+  pull-requests: write
+jobs:
+  CLAAssistant:
+    runs-on: ubuntu-latest
+    steps:
+      - id: fetch-token
+        name: Fetch installation token
```

```suggestion
      - name: Fetch installation token
        id: fetch-token
```

nit: we usually put `name` ahead of `id` for readability